src/drag-drop-polyfill.ts

```diff
@@ -1387,6 +1387,16 @@ module DragDropPolyfill {
             transitionEndCb();
             return;
         }
+        // add class containing transition rules
+        dragImage.classList.add( CLASS_DRAG_IMAGE_SNAPBACK );
+
+        const csDragImage = getComputedStyle( dragImage );
+        const durationInS = parseFloat( csDragImage.transitionDuration );
+        if (durationInS === NaN || durationInS ====0) {
```

Comment:
please use isNaN() because NaN is quite special, see https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/isNaN

`durationInS ====0` is not valid, please replace by `durationInS === 0`